### PR TITLE
Added proper ice damage sound

### PR DESCRIPTION
**Pull request by Kasterisk**
**Brief Description of What This PR Does**

Added the ice damage sound made by TwitchyWhalez. Had to add a new `damageType` property to the chunks json for distinguishing damage types.

**Related Issues**

Finishes part of #1609

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: simulation_parameters/microbe_stage/biomes.json [hunk 1]
```diff
@@ -1559,6 +1568,7 @@
                 },
                 "iceShard": {
                     "name": "ICESHARD",
+                    "identifier": "ice",
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/biomes.json:1571]: maybe "damageType" would be a better name here?

Comment (Kasterisk) [reply simulation_parameters/microbe_stage/biomes.json:1571]: Sounds good.

File: src/microbe_stage/FloatingChunk.cs [hunk 2]
```diff
@@ -125,6 +130,7 @@ public class FloatingChunk : RigidBody, ISpawned, ISaveLoadedTracked
         Size = chunkType.Size;
         Damages = chunkType.Damages;
         DeleteOnTouch = chunkType.DeleteOnTouch;
+        DamageType = string.IsNullOrEmpty(chunkType.DamageType) ? "chunk" : chunkType.DamageType;
```
Comment (hhyyrylainen) [inline src/microbe_stage/FloatingChunk.cs:133]: Doesn't this need handling in `CreateChunkConfigurationFromThis` as well? That seems like a bug just waiting to strike to me.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
The code seems mostly fine to me, just one small comment.
Also just to confirm this probably works with older saves due to the string.IsNullOrEmpty check and defaulting to "chunk", right?

**Comment by Kasterisk** [comment]
Yeah, all chunks ended up having the same damage sound when you load older saves.

File: src/microbe_stage/FloatingChunk.cs (hunk 2 above)
Comment (Kasterisk) [reply src/microbe_stage/FloatingChunk.cs:133]: Done, forgot about that.

**Review by hhyyrylainen — APPROVED** [review]
I'm now happy with the code implementation. Needs playtesting / listening next.

**Comment by hhyyrylainen** [comment]
I guess people with ears still don't want to help me out, so I'll just merge this.

**Pull request merged**